crates/sp-domains/src/lib.rs

```diff
@@ -1020,6 +1020,9 @@ sp_api::decl_runtime_apis! {
 
         /// Returns the execution receipt hash of the given domain and domain block number
         fn receipt_hash(domain_id: DomainId, domain_number: HeaderNumberFor<DomainHeader>) -> Option<HeaderHashFor<DomainHeader>>;
+
+        /// Reture the domain transaction byte fee
+        fn domain_transaction_byte_fee() -> Balance;
```

Comment:
I'm also confused with this naming. Shouldn't this be consensus storage fee instead? Domain transaction byte fee is very specific to storage cose of the externalities during the state trnasition on the domain itself

Reply:
The `domain_transaction_byte_fee` is 2x of the consensus storage fee that the consensus chain charged for the domain tx in the bundle.

>Domain transaction byte fee is very specific to storage cose of the externalities during the state trnasition on the domain itself

That cost is defined by the domain chain, not querying from the consensus chain.

Reply:
Okay this is a confusing name. I would say the following APIs would be easier to understand
```
fn consensus_chain_storage_fee()
fn domain_storage_fee_multiplier()
fn domain_extrinsic_validation_multipler() // since we are planning to have higher multipler while validating txns in the txpool
```
thoughts ?